### DISCUSSION TITLE
How will be approved

### DISCUSSION THREAD

**Discussion by Dipanshu0984** [community/community#131291] [GitHub Education] [answered]
I had applied for student id and its not approved till now how much time it will take

**Comment by davevad93** [comment]
Hi @Dipanshu0984 ,

It usually takes a few days, but may take longer depending on different factors such as the volume of applications.

When the benefits are available, the status label will turn purple and display the following:

- `Congrats! Your request for @Dipanshu0984 was approved and your benefits are now available`

**Reply by Dipanshu0984** [reply]
After approval how much times it takes to give us benifits

**Reply by davevad93** [reply, marked as answer]
Hey @Dipanshu0984 , in most cases benefits are available within 2 weeks from approval, if you were approved in July and are still waiting for your benefits, that's not normal. I remember a discussion months ago where a user claimed he gained access to his benefits 3 months after approval, but again that's unusual, it's usually within a couple of weeks.

**Reply by Dipanshu0984** [reply]
My id was approved how much time it will take to give the service to get all packs

**Reply by davevad93** [reply]
The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @Dipanshu0984 was approved and your benefits are now available.`

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)
